src/main/scala/com/datastax/cdm/job/DiffData.scala

```diff
@@ -29,7 +30,10 @@ object DiffData extends BasePartitionJob {
     if (!parts.isEmpty()) {
       originConnection.withSessionDo(originSession =>
         targetConnection.withSessionDo(targetSession =>
-          jobFactory.getInstance(originSession, targetSession, propertyHelper).initCdmRun(runId, prevRunId, parts, trackRunFeature, JobType.VALIDATE)));
+          jobFactory.getInstance(originSession, targetSession, propertyHelper).initCdmRun(runId, prevRunId, parts, trackRunFeature, jobType)));
+      var ma = new CDMMetricsAccumulator(jobType)
```

Comment:
should we move these 2 lines to the `BasePartitionJob` class instead? And, remove from these 3 class files?